vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -471,15 +471,25 @@ WinRTHttpResource::PerformSendRequest(HttpMethod &&method, Uri &&rtUri, IInspect
       auto inputStream = co_await response.Content().ReadAsInputStreamAsync();
       auto reader = DataReader{inputStream};
 
-      // #9510 - 10mb limit on fetch
-      co_await reader.LoadAsync(10 * 1024 * 1024);
+      // #9510 - We currently accumulate all incoming request data in 10MB chunks
+      uint32_t segmentSize = 10 * 1024 * 1024;
```

Comment:
>10 * 1024 * 1024 [](http://example.com/codeflow?start=29&length=16)

nit: I wonder if there's a custom literal like

auto operator""_MB(uint32_t x) -> uint32_t { return 1024*1024*x; }

... that would enable writing this as

uint32_t segmentSize = 10_MB;